openmls/src/tree/mod.rs

```diff
@@ -776,6 +776,11 @@ impl RatchetTree {
         // Determine if Commit needs a path field
         let path_required = has_updates || has_removes || !has_adds;
 
+        // If members were removed, truncate the tree.
+        if has_removes {
+            self.trim_tree()
```

Comment:
This needs a test.

Reply:
Ugh. I really should have thought of that. Fixing.

Reply:
I really blame codecov. It was so green I didn't even think of adding any tests 😅